reconcile/terraform_resources.py

```diff
@@ -250,11 +250,9 @@ def populate_oc_resources(spec, ri):
         logging.error(msg)
 
 
-def fetch_current_state(dry_run, namespaces, thread_pool_size,
+def fetch_current_state(namespaces, thread_pool_size,
                         internal, use_jump_host):
     ri = ResourceInventory()
-    if dry_run:
```

Comment:
+1 to reading stuff even under dry_run. :)

Reply:
reverting this as we can't really compare the diffs in secrets in dry-run mode (they rely on the terraform state)